### PR TITLE
Fix about section display on Safari

### LINKED ISSUE #57: Layout off on iOS
Hi, loving the design so far. Keep up the good work 👍 

Layout is not working on iPhone, works fine in android.

IMG_6A2482F03F2A-1

Would also love to get proper categories and tags support but I don't want to burden you with stuff 😃

### PR DESCRIPTION
**Pull request by DotIN13**
Fix #57

### CODE REVIEW

File: _sass/search.scss [hunk 1]
```diff
@@ -115,7 +111,7 @@ html, body {
     .search-section {
       border-bottom-left-radius: 0;
       border-bottom-right-radius: 0;
-      box-shadow: 0 1px 2px 0 rgba(32,33,36,0.28);
+      box-shadow: 0 1px 6px 0 rgba(32,33,36,0.28);
```
Comment (thelehhman) [inline _sass/search.scss:114]: Hey, I don't think we need the shadow changes.

**Comment by thelehhman** [comment]
Looks good. Just remove the tweaks to `box-shadow` and we should be good to go!

**Comment by DotIN13** [comment]
Sorry my mistake to include such irrelevant stuff in this. I've fixed that and please check it again!

But do tell me how you feel on the drop shadow cuz the original version does not look good on mobile devices. If you prefer, I'll do another PR.

**Comment by thelehhman** [comment]
Now worries. Thanks!

**Comment by DotIN13** [comment]
On double check I realize you personally made the changes to the drop shadow, I originally thought it was my mistake. Then all good, we'll leave it be as you like!

**Pull request merged**